src/OrchardCore/Orchard.Environment.Shell/RunningShellTable.cs

```diff
@@ -126,9 +133,9 @@ private string GetHostAndPrefix(string host, string appRelativePath)
 
         }
 
-        private string GetHostAndPrefix(ShellSettings shellSettings)
+        private string[] GetHostsWithPrefix(ShellSettings shellSettings)
```

Comment:
How often is it called per request per app lifetime? Concern is perf. If only called once then GTG.